Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -382,6 +414,25 @@ final class YearInReviewCoordinator: NSObject, Coordinator {
                             hideDonateButton: true)
                     }
                 }
+            case .savedCount:
+                if slide.display == true,
+                   let data = slide.data {
+                    let decoder = JSONDecoder()
+                    if let savedSlidedata = try? decoder.decode(SavedArticleSlideData.self, from: data),
+                       savedSlidedata.savedArticlesCount > 3,
+                       savedSlidedata.articleTitles.count > 2 {
```

Comment:
Should this be `> 3`? I'm curious about the difference here. Also minor casing issue: `savedSlideData`.